apigw/config/certificates/keycloak-docker-dev.pem

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Add description of this certificate to `apigw/config/certificates/README.markdown`. Is this a default certificate in all keycloak installations, or a specific one somewhere in our config that gets imported?

Or...if this certificate is not used at all, it should be removed

Reply:
I'll remove it for now, lets add a new one once we take the signing in use